docs/pages/configuration/pipelines/functions/build_images.mdx

```diff
@@ -5,28 +5,54 @@ sidebar_label: build_images
 
 The function `build_images` builds either all images (`--all` flag) or the images provided as arguments (e.g. `build_images api ui`). Images are, by default, processed in parallel.
 
+## Arguments
 
-## Args
+### `--image`
 
+The image you want to have built.
+
+### Usage
+
+```
+build_image "image1" "image2"
+
+```
 
 ## Flags
 
+By default, flags are disabled. If a flag is enabled, the following action is taken.
+
 ### `--all`
 
-### `--tag / -t`
+All images will be built.
 
-### `--force-rebuild`
+### `--tag / -t`
 
-### `--max-concurrent`
+The build image tag will be returned.
 
-### `--sequential`
+### `--skip-build`
 
-### `--skip`
+The image will not be built.
 
 ### `--skip-push`
 
+The image will not be pushed to a container.
+
 ### `--skip-push-on-local-kubernetes`
 
+The image will not be pushed to a local kubernetes cluster.
+
+### `--force-rebuild`
+
+The images will be rebuilt.
+
+### `--sequential`
+
+Images will be built in sequential order, one after another.
+
+### `--max-concurrent-builds`
+
+An integer that represents the maximum number of images that can be built concurrently.
 
 ## Config Overwrite Flags
 
```

Comment:
Please remove the content under here and make a note to add the config overwrite flag fragment.